server/tests/article.test.js

```diff
@@ -15,9 +15,9 @@ const user = {
 };
 
 const article = {
-  title: faker.random.words(),
-  body: faker.random.words(),
-  description: faker.random.words()
+  title: faker.lorem.sentence(),
+  body: faker.lorem.paragraph(),
+  description: faker.lorem.sentences()
 };
```

Comment:
I changed the random words generated by faker to be generated from lorem sentences. This is because the tests were failing since the random words may occasionally be less than the required minimum length.